testing/bat/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Chloe Kudryavtsev <toast@toastin.space>
+# Maintainer: Chloe Kudryavtsev <toast@toastin.space>
+pkgname=bat
+pkgver=0.11.0
+pkgrel=0
+pkgdesc="A cat(1) clone with wings"
+url="https://github.com/sharkdp/bat"
+arch="x86_64" # limited by rust/cargo
+license="Apache-2.0"
+makedepends="cargo"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/sharkdp/bat/archive/v$pkgver.tar.gz"
+
+export CARGO_HOME="$srcdir"/cargo
+export RUSTFLAGS="-C target-feature=-crt-static"
```

Comment:
Does our Rust not do this by default? I mean that shouldn't block this PR, but means I'd have to adjust #8628